mxcubeweb/routes/lims.py

```diff
@@ -28,7 +28,9 @@ def proposal_samples():
         try:
             res = jsonify(app.lims.synch_with_lims())
         except Exception as ex:
-            logging.getLogger("MX3.HWR").exception()
+            logging.getLogger("MX3.HWR").exception(
+                "Could not synchronize with LIMS %s" % str(ex)
+            )
```

Comment:
The `exception` method should add the exception and its traceback automatically. Unless this is unwanted, it should be okay to simplify to:

```suggestion
            logging.getLogger("MX3.HWR").exception(
                "Could not synchronize with LIMS"
            )
```

https://docs.python.org/3/library/logging.html#logging.exception

Reply:
I expected a comment like that from you @fabcor-maxiv  :hugs:
but when reading the logs i prefer having the message and the exception together in a line. 
it's like error msg follow by reason in one line.  but you right, my point is just aesthetic.

Reply:
Then it's fine : )

The logging is all over the place in MXCuBE. I still have hope that one day we can clean up all the logging.